index.html

```diff
@@ -61,20 +62,24 @@ <h2>My List</h2>
 
         <!-- empty item -->
         <div class="empty-item" style="display:none">
-          <div class='card shadow p-4 mb-4 bg-white'>
-            <div class='card-body'>
-              <div class="float-start">
-                <div class="form-check">
-                  <input id="" class="empty-checkbox form-check-input" type="checkbox"
-                    onclick="toggleCheckboxText()"><label class="empty-label form-check-label" for=""></label>
+          <div class='card shadow p-1 mb-1 bg-white'>
+            <div class='card-body'>          
+              <div class="row">
+                <div class="col-sm-11">
+                  <div class="form-check">
+                    <input id="" class="empty-checkbox form-check-input" type="checkbox"
+                      onclick="toggleCheckboxText()"><label class="empty-label form-check-label" for=""></label>
+                  </div>
+                </div>
+                <div class="col-sm-1">  
+                      <button class="btn btn-primary" onclick="deleteTask()"><span
+                          class="d-none d-sm-block">Delete</span>
+                        <i class="fa-regular fa-trash-can d-block d-sm-none"></i>
+                      </button>
+                    </div>
+                  </div>
                 </div>
               </div>
-              <div class="float-end">
```

Comment:
you want to keep the div class="float-end" here to ensure that the button is set to the far most right